.circleci/config.yml

```diff
@@ -1147,6 +1146,28 @@ jobs:
           event: fail
           template: basic_fail_1
 
+  check-generated-mocks-op-node:
+    docker:
+      - image: us-docker.pkg.dev/oplabs-tools-artifacts/images/ci-builder:latest
```

Comment:
CI builder doesn't have `mockery` installed

Reply:
Yep: https://github.com/ethereum-optimism/optimism/pull/5839#issuecomment-1570398060

I can install it in the `check-generated-mocks-op-node` job before running generate, or alternatively add it to the `ci-builder` container. I'll see if I (outside collaborator) can update the docker container and if CI publishes the images.

Reply:
I opened #5844 to add mockery to the `ci-builder` container. Once that PR is good to go and merged, we can re-run CI in this PR.